.github/workflows/CI.yml

```diff
@@ -0,0 +1,63 @@
+name: CI
+
+on:
+  # manual trigger for every branch
+  workflow_dispatch:
+  # push on master and PR merge on master
+  push:
+    branches: [ master ]
+  # PRs (first and subsequent pushes)
+  pull_request:
+
+jobs:
+  test-suite:
+    runs-on: ubuntu-latest
+    steps:
+      # ===== slack start =====
+      - name: slack tests started
+        uses: slackapi/slack-github-action@v1.16.0
+        with:
+          payload: "{\"text\": \":test_tube: ${{ github.repository }} - Tests started \",\"blocks\": [{\"type\": \"section\",\"text\": {\"type\": \"mrkdwn\",\"text\": \":test_tube: *${{ github.repository }}* - Tests <${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}|${{ github.run_number }}> started by *${{ github.actor }}* on branch *${{ github.ref }}*\"}}]}"
```

Comment:
[minor] Maybe it would be better if you break payloads into multiple lines. It's easier to read them.

Reply:
It seems that we cannot split the messages to multiple lines. 

https://github.com/skroutz/analytics.js/actions/runs/1558739229

Reply:
my 2 cents, I'm a bit inclined to keep these as they are in a single line anyway